Automattic/wp-calypso
# Add tests for Calypso server

#### Changes proposed in this Pull Request

Adds a bunch of unit tests for the main app of Calypso server.

The main driver for this change is to have some safety-net in case we want to change how the server works (which I'll likely do very soon to improve its performance, monitoring and devloop)

#### Testing instructions

* Verify test passes in CI.
* Optionally, run it locally with `yarn test-server ./client/server/pages/test/index.js`) and verify it doesn't output anything unexpected in the console and jest doesn't complain about open handles:

![image](https://user-images.githubusercontent.com/975703/84357632-4bb13380-abc6-11ea-83f0-7f7cc7a0a0dc.png)
